src/components/Page.js

```diff
@@ -37,19 +41,26 @@ export default class Page extends Component {
           pageText,
           pageNumber,
           activeClass,
+          itemClass,
+          linkClass,
           disabledClass,
           isActive,
           isDisabled,
         } = this.props;
 
         const css = cx({
           [activeClass]: isActive,
-          [disabledClass]: isDisabled
+          [disabledClass]: isDisabled,
+          [itemClass]: itemClass !== undefined,
```

Comment:
`classnames` already ignores falsy values, so you can just pass `itemClass` like this:
```javascript
const css = cx(itemClass, {
  [activeClass]: isActive,
  [disabledClass]: isDisabled
});
```

You can see further information about this here: https://github.com/JedWatson/classnames#usage